starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,8 @@
 <h1>
   {{title}}
 </h1>
+
+<a [routerLink]="['home']"> Home  </a>
+<a [routerLink]="['movie', movie]"> Movies </a>
```

Comment:
`<a [routerLink]="['movie', movie]"> Movies </a>`
Should be on the HomePage passing the id of each movie with `movie.id`